source/docs/hardware/hardware-basics/status-lights-ref.rst

```diff
@@ -6,202 +6,511 @@ Many of the components of the FRC Control System have indicator lights that can
-
-Flashing Red/Green indicate ready for calibration. Several green flashes indicates successful calibration, and red several times indicates unsuccessful calibration.
-
-Talon-SRX speed controllers
----------------------------
-
-.. image:: images/status-lights/talonSRXLight.png
++-----------+----------+------------------------------------------------------------------------+
+| Green     | Solid    | Full forward output                                                    |
+|           +----------+------------------------------------------------------------------------+
+|           | Blinking | Proportional to forward output voltage                                 |
++-----------+----------+------------------------------------------------------------------------+
+| Red       | Solid    | Full reverse output                                                    |
+|           +----------+------------------------------------------------------------------------+
+|           | Blinking | Proportional to forward output voltage                                 |
++-----------+----------+------------------------------------------------------------------------+
+| Orange    | Solid    | FRC robot disabled, PWM signal lost, or                                |
+|           |          | signal in deadband range (+/- 4% output)                               |
++-----------+----------+------------------------------------------------------------------------+
+| Red/Green | Blinking | Ready for calibration. Several green flashes indicates successful      |
+|           |          | calibration, and red several times indicates unsuccessful calibration. |
++-----------+----------+------------------------------------------------------------------------+
+
+Talon SRX speed controller
+--------------------------
+
+.. image:: images/status-lights/talon-srx-status-lights.png
+  :width: 600
+
++-------------------------------------------------+
+| Blink Codes During Calibration                  |
++========================+========================+
+| Status LEDs Blink Code | Talon SRX State        |
++------------------------+------------------------+
+| Flashing Red/Green     | Calibration Mode       |
++------------------------+------------------------+
+| Blinking Green         | Successful Calibration |
++------------------------+------------------------+
+| Blinking Red           | Failed Calibration     |
++------------------------+------------------------+
+
++--------------------------------------------------------------------------------------------+
+| Blink Codes During Normal Operation                                                        |
++==============================+================+============================================+
+| LEDs                         | Colors         | Talon SRX State                            |
```

Comment:
Make this row the header row and move the row above to be a section title

Reply:
Fixed in c172d27.